richtext-commonmark/src/commonMain/kotlin/com/halilibo/richtext/markdown/node/AstNodeLinks.kt

```diff
@@ -14,7 +14,7 @@ public data class AstNodeLinks(
   var next: AstNode? = null
 ) {
 
-  internal inline fun <reified T : AstNodeType> firstParentOrNull(): T? {
+  public inline fun <reified T : AstNodeType> firstParentOrNull(): T? {
```

Comment:
It's not strictly required but exposing all of the types for the override makes sense. It would almost surely have been done if there was inline content override like we have added.